components/Spotlight/SpotlightPreview.js

```diff
@@ -0,0 +1,33 @@
+import React from "react"; 
+import styles from "./Spotlight.module.css"
+
+class SpotlightPreview extends React.Component{
+    render(){
+        return (
+            <table className="large tableContent" cellSpacing="0">
```

Comment:
In my experience, tables come with some really challenging side effects. I'm happy to relent on the use here since I suppose we're literally just building a table with two-columns per row... but past me is having flashbacks for bad times come future maintenance.

Reply:
haha I admit this was a pretty lazy implementation. A lot of the designs I went for on this page was playing on the safe side - trying to re-use current colors, ideas, and styles to keep a consistency in design. A by product of this thinking resulted in me re-using the tables and css from the home page.

We could definitely strip the table from the this. Do you think something like [grids](https://react-bootstrap.github.io/layout/grid/#row-layout-col-sizing) would be better? Or like you said in the overall feedback, maybe this would change if we moved toward another implementation where these wouldn't be present.